src/components/wizards/new-vm/NewVmWizard.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Wizard, Button } from 'patternfly-react';
+import { CreateVmWizard } from '../create-vm/CreateVmWizard';
+import { ImportVmWizard } from '../import-vm/ImportVmWizard';
+import './NewVmWizard.css';
+
+export class NewVmWizard extends React.Component {
+
+    state = {
+      createVM: false,
+      importVM: false
+    }
+  
+    openCreateVmWizard = () => this.setState({createVM: true});
+
+    openImportVmWizard = () => this.setState({importVM: true});
+  
+    render() {
+      let wizard;
+      if(this.state.createVM) {
+        wizard = <CreateVmWizard onHide={this.props.closeWizard} {...this.props}/>
+      } else if(this.state.importVM) {
+        wizard = <ImportVmWizard onHide={this.props.closeWizard} {...this.props}/>
+      }
+      return (
+        <React.Fragment>
+          <Wizard show onHide={this.props.closeWizard}>
+            <Wizard.Header title="Create Virtual Machine" onClose={this.props.closeWizard} />
+            <Wizard.Body>
+              <Wizard.Row>
+                <Wizard.Main>
+                  <Wizard.Contents stepIndex={0} activeStepIndex={0} className="wizard-content">
+                    <button className="btn btn-link wizard-button " onClick={this.openCreateVmWizard}>
```

Comment:
You could make a component out of this

Reply:
there is a pf-react Button Component?

Reply:
I will use Button from pf-react and also extract the whole 'button with icon and label' to separate component

Reply:
Yes, let's reuse patternfly-react's `Button` if possible.

Reply:
Moved to separate component `ButtonWithIcon` which is using `Button`, `Icon` and `Col` pf-react components